core/src/data_sources/data_source.rs

```diff
@@ -142,6 +142,7 @@ pub struct Document {
     pub title: String,
     pub mime_type: String,
     pub tags: Vec<String>,
+    pub parent_id: Option<String>,
```

Comment:
why diff this file for now?

Reply:
don't we want to return the parent_id here ?

Reply:
seen irl with @philipperolet , to be consistent we want to add parent_id in return too